_life/renting_legal.md

```diff
@@ -0,0 +1,73 @@
+Legal aspects of renting in Switzerland
+=======================================
```

Comment:
Instead of making this a separate page, I propose adding it to the "Accommodation" page. What do you think?